lib/src/model/package.dart

```diff
@@ -14,6 +14,10 @@ import 'package:pub_semver/pub_semver.dart';
 
 final RegExp substituteNameVersion = RegExp(r'%([bnv])%');
 
+// Unlikely to be mistaken for an identifier, html tag, or something else that
+// might reasonably exist normally.
+final String HTMLBASE_PLACEHOLDER = '\%\%HTMLBASE\%\%';
```

Comment:
This was just off the top of my head, happy to replace it with something else.
The backslash escapes are unnecessary, but should prevent this string from being substituted if someone runs Dartdoc on Dartdoc itself.

Reply:
This string is short enough I can imagine someone else using it reasonably.  Maybe:  ```'\%\%__HTMLBASE_dartdoc_internal_only\%\%'``` instead?